modules/file-system/filestore/variables.tf

```diff
@@ -34,8 +34,13 @@ variable "region" {
   type        = string
 }
 
-variable "network_name" {
-  description = "The name of the GCE VPC network to which the instance is connected."
+variable "network_id" {
+  description = <<-EOT
+    The name of the GCE VPC network to which the instance is connected. 
+    If using shared VPC, full network id must be given with format:
+    `projects/<project_id>/global/networks/<network_name>`"
+    Plain network name is accepted for non-shared VPC case.
```

Comment:
"Plain network name is accepted" does not appear to be the case in the code. Empirically, this blueprint:

```yaml
blueprint_name: simple

vars:
  project_id:  ## Set project id here
  deployment_name: simple
  region: us-central1
  zone: us-central1-b

# Documentation for each of the modules used below can be found at
# https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/README.md

deployment_groups:
- group: first
  modules:
  - id: network1
    source: modules/network/pre-existing-vpc
  - id: simpelfs
    source: modules/file-system/filestore
    settings:
      network_id: default
  - id: simplevm
    source: modules/compute/vm-instance
    use:
    - network1
    settings:
      machine_type: e2-medium
      add_deployment_name_before_prefix: true
```

yields the following error

```
╷
│ Error: Invalid index
│
│   on modules/filestore/main.tf line 42, in locals:
│   42:   network_name     = local.split_network_id[4]
│     ├────────────────
│     │ local.split_network_id is list of string with 1 element
│
│ The given key does not identify an element in this collection value.
╵
╷
│ Error: Invalid index
│
│   on modules/filestore/main.tf line 43, in locals:
│   43:   network_project  = local.split_network_id[1]
│     ├────────────────
│     │ local.split_network_id is list of string with 1 element
│
│ The given key does not identify an element in this collection value: the given index is greater than or equal to the length of the collection.
```

Reply:
This was my intention (as it is how the code is written). I forgot to update the description. Good catch.

I will update description and add validation.

Reply:
Done.